src/Calendar/Calendar.test.js

```diff
@@ -135,7 +133,7 @@ describe('<Calendar />', () => {
 
     // check that April was selected
     const currentDateDisplayed = wrapper.state('currentDateDisplayed');
-    expect(currentDateDisplayed.getFullYear()).toEqual(2021);
+    expect(currentDateDisplayed.getFullYear()).toEqual(2022);
```

Comment:
:D I also fixed this in my PR

Reply:
I actually have a real fix, so that this problem doesn't happen next year, in my update unit test branch that i will push after your changes are merged 😄